server/src/pages/shot/view.js

```diff
@@ -312,15 +312,15 @@ class Body extends React.Component {
 
   renderExtRequired() {
     return <div className="default-color-scheme notification">
-      <div> Page Shot is an experimental extension for Firefox. <a href={ this.props.backend } onClick={ this.clickedInstallExtension.bind(this) }>Get it here</a></div>
-      <a className="close" onClick={ this.closeGetPageshotBanner.bind(this) }></a>
+      <div> Firefox Screenshots is an experimental extension for Firefox. <a href={ this.props.backend } onClick={ this.clickedInstallExtension.bind(this) }>Get it here</a></div>
```

Comment:
I thought about removing "for Firefox", but it seemed potentially confusing given future goals.  But yeah, it'll probably get overwritten later.